dockerfiles/Dockerfile

```diff
@@ -0,0 +1,67 @@
+### This is a full chipyard setup
+
+# BUILD BASE FOR CI
+
+FROM ubuntu:18.04 as base
+ARG CHIPYARD_HASH
+
+MAINTAINER https://groups.google.com/forum/#!forum/chipyard
+
+# Install dependencies for ubuntu-req.sh
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+               curl \
+               git \
+               sudo \
+               ca-certificates \
+               keyboard-configuration \
+               console-setup
```

Comment:
Should this be finished with a `rm -rf /var/lib/apt/lists/*` as well?

Reply:
I tried that, but then when ubuntu-req.sh was run during the build, it wasn't able to find any of the packages.